dist/htmlminifier.js

```diff
@@ -1162,6 +1162,14 @@
     return text;
   }
 
+  function uniqueId(value) {
```

Comment:
By replacing `<!-- htmlmin:temp -->` with `<!--!deadb33f-->`, we also get to simplify `isIgnoredComment()` would may give a minor performance boost.

Also, this PR avoids any extra work done when `ignoreCustomFragments` or `<!-- htmlmin:temp -->` are not specified in the input, i.e. `uniqueId()` and `String.prototype.replace()` would never be called if none of this features are requested by the input.